src/app/core/components/footer/footer.component.html

```diff
@@ -1,4 +1,17 @@
-<footer class="govuk-footer" role="contentinfo">
+ <footer role="contentinfo">
```

Comment:
```suggestion
 <footer class="govuk-footer" role="contentinfo">
```

Removing this class breaks the font-family of the bottom-most text.